eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgAuditDb.scala

```diff
@@ -52,15 +53,25 @@ class PgAuditDb(implicit ds: DataSource) extends AuditDb with Logging {
         statement.executeUpdate("CREATE INDEX relayed_trampoline_payment_hash_idx ON relayed_trampoline(payment_hash)")
       }
 
+      def migration56(statement: Statement): Unit = {
+        statement.executeUpdate("ALTER TABLE sent               ALTER COLUMN timestamp SET DATA TYPE TIMESTAMP WITH TIME ZONE USING timestamp with time zone 'epoch' + timestamp * interval '1 millisecond'")
```

Comment:
This was provided as an example in the [postgres documentation(https://www.postgresql.org/docs/9.6/sql-altertable.html).

Reply:
The idea was to make similarities obvious between lines. Removed in 641abae.

Reply:
It's still there in this file...?

Reply:
Oops, forgot one file. Fixed with 1ff9b7b.